<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatenewDialogBase</class>
 <widget class="QDialog" name="CreatenewDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>198</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>130</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Create new land use layer</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>518</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="1" colspan="2">
     <widget class="QComboBox" name="selectIDbuildingCombo"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Floors: </string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="8" column="2">
     <widget class="QPushButton" name="pushButtonLUNewFileDLG">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="SavelocationlabelLU">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Select save Location:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="LUincGFcheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Ground floor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="LUincLFcheckBox">
        <property name="text">
         <string>Lower floor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="LUincUFcheckBox">
        <property name="text">
         <string>Upper floor</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="pushButtonSelectLocationLU">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="createNewLUFileCheckBox">
      <property name="text">
       <string>Use building layer:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QPushButton" name="closePopUpLUButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEditLU">
      <property name="placeholderText">
       <string>[Save to memory file]</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QComboBox" name="selectbuildingCombo"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Select ID Column</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QRadioButton" name="lu_postgis_radioButton">
      <property name="text">
       <string>postgis</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="lu_memory_radioButton">
        <property name="text">
         <string>memory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="lu_shp_radioButton">
        <property name="text">
         <string>shapefile</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
